calculator.js

```diff
@@ -0,0 +1,25 @@
+function
```

Comment:
This would trigger a syntax error, because a function has to be followed by the parenthesis (for arguments) and brackets (for the code).